### DISCUSSION TITLE
How can i access all the tools?

### DISCUSSION THREAD

**Discussion by maxmil** [bk138/gromit-mpx#217] [Q&A] [answered]
I've just got a new Wacom tablet and am looking for a screen annotation tool. Gromit looks just like what I'm after. I'd like to be able to switch easily between quite a few different tools (pens with different colors and sizes, smoothed tools, orthogonal tools, arrows....) however with just the CONTROL, SHIFT and META/ALT keys I don't have many combinations.

I'm using a pen that has 1 button but this is not comfortable to operate.

Am I missing something? Is there another way to activate tools other than via the keyboard shortcuts configured in the config file?

**Comment by bk138** [comment, marked as answer]
Hi @maxmil, thanks for reaching out! What you're seeing is the current state of affairs. That said, the tool is not done yet, there's still functionality like this to add.
